scss/modules/modals/_base.scss

```diff
@@ -88,3 +88,32 @@
     background-color: $color-shade-dark;
   }
 }
+
+.c-fullscreen_modal {
+  &__content {
+    background: $color-base;
+  }
+
+  &__header {
+    background: $color-base;
+  }
+
+  &__body {
+    color: $base-font-color;
+  }
+
+  &__back,
+  &__close {
+    color: rgba($base-font-color, 0.5);
+
+    &:hover {
+      // background: $color-shade-lightest;
```

Comment:
Please remove cruft

Reply:
Sorry, I meant: please remove commented out styles. It looks like these two locations haven't changed since the last time I reviewed?